pkg/broker/filter/filter_handler.go

```diff
@@ -152,6 +153,37 @@ func NewHandler(logger *zap.Logger, tokenVerifier *auth.OIDCTokenVerifier, oidcT
 	}, nil
 }
 
+func getOIDCIdentity(r *http.Request) (string, error) {
```

Comment:
In the pkg/auth package we have some methods which can help you with this. You probably only need to add a new one, which is similar to the [`OIDCTokenVerifier.VerifyRequest()`](https://github.com/knative/eventing/blob/ff37e4e2fc0fbe5438a8369dcd704d3235f9d501/pkg/auth/token_verifier.go#L85-L101), but does not check on the given policy refs, but checks only if a given OIDCSubject matches